stdlib/Random/test/runtests.jl

```diff
@@ -2,6 +2,7 @@
 
 using Test, SparseArrays
 using Test: guardseed
+using Statistics: mean
```

Comment:
Does this require `Statistics` to be added to the test dependencies in `Project.toml`?

Reply:
Good catch, thanks